draft-ietf-rats-msg-wrap.md

```diff
@@ -652,7 +652,7 @@ The following subsections provide further elaboration on these points, particula
 
 CMW itself does not provide any mechanisms for authenticity, integrity protection, or confidentiality.
 It is the responsibility of the designer for each use case to determine the necessary security properties and implement them accordingly.
-In some scenarios, a secure channel (e.g., via TLS {{?BCP195}}) or object-level security (e.g., using JWT {{?BCP225}}) may be sufficient, but this is not always the case.
+In some scenarios, for instance, if CMW encapsulates Reference Values and/or Endorsements, a secure channel (e.g., via TLS {{?BCP195}}) or object-level security (e.g., using JWT {{?BCP225}}) may suffice, but this is not always the case.
```

Comment:
```suggestion
In some scenarios, for instance, if CMW encapsulates Reference Values or Endorsements, a secure channel (e.g., via TLS {{?BCP195}}) or object-level security (e.g., using JWT {{?BCP225}}) may suffice, but this is not always the case.
```

Reply:
CMW can be signed. Wouldn't signed CMW address integrity protection and authenticity without needing TLS?

Reply:
Given TLS addresses confidentiality, replay, as well as integrity and authenticity. What other threat is remaining?